MsixCore/msixmgr/msixmgr.cpp

```diff
@@ -192,26 +195,79 @@ int main(int argc, char * argv[])
     CommandLineInterface cli(argc, argv);
 
     const HRESULT hrCreateRequest = cli.Init();
+
+    // Initializing all the variables for Telemetry
+    std::wstring WorkflowId, SourceApplicationId, CorrelationId, ErrorCode = L"", ErrorDesc = L"";

```

Comment:
Variable casing needs to be fixed here.
Ex: workflowId instead of WorkflowId and correlationId instead of CorrelationId.

Reply:
Done